CONTRIBUTING.md

```diff
@@ -19,13 +24,18 @@ Being a contributor at The Algorithms, we request you to follow the points menti
 - Your work will be distributed under the [MIT License](https://github.com/TheAlgorithms/Go/blob/master/LICENSE) once your pull request has been merged.
 - Please follow the repository guidelines and standards mentioned below.
 
-**New implementation** New implementations are welcome!
+#### New implementation
 
-You can add new algorithms or data structures that are **not present in the repository** or that can **improve** the old implementations (**documentation**, **improving test cases**, removing bugs, or in any other reasonable sense)
+- New implementations are welcome!
 
-**Issues** Please avoid opening issues asking to be "assigned” to a particular algorithm. This merely creates unnecessary noise for maintainers. Instead, please submit your implementation in a pull request, and it will be evaluated by project maintainers.
+- You can add new algorithms or data structures that are **not present in the repository** or that can **improve** the old implementations (**documentation**, **improving test cases**, removing bugs, or in any other reasonable sense)
+
+#### Issues
+
+- Please avoid opening issues asking to be "assigned” to a particular algorithm. This merely creates unnecessary noise for maintainers. Instead, please submit your implementation in a pull request, and it will be evaluated by project maintainers.
 
 ### Making Changes
+---
```

Comment:
GitHub adds horizontal lines here, but it's supposed to be a level 2 heading

Reply:
this is intentional, i personally find it easier to read the document this way. if you disagree, then I can remove it.